lib/cocoapods/installer/analyzer.rb

```diff
@@ -437,7 +437,7 @@ def generate_aggregate_targets(resolver_specs_by_target, target_inspections)
             aggregate_target.merge_embedded_pod_targets(embedded_pod_targets)
           end
         end
-        aggregate_targets
+        [aggregate_targets, pod_targets]
```

Comment:
the methods name and return type does not match and the documentation would need update as well.